ee/zentral/contrib/intune/api_views.py

```diff
@@ -0,0 +1,44 @@
+from django.shortcuts import get_object_or_404
+from django.urls import reverse
+from rest_framework import generics, status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from zentral.core.events.base import EventRequest
+from zentral.utils.drf import DefaultDjangoModelPermissions, DjangoPermissionRequired
+from .models import Tenant
+from .serializers import TenantSerializer
+from .tasks import sync_inventory
+
+
+class TenantList(generics.ListAPIView):
```

Comment:
Implement the other operation so ListCreate, not only read only. Also, use the [ListCreateAPIViewWithAudit](https://github.com/zentralopensource/zentral/blob/6e63568a1ebf9a77b01f6cc04a96e68509a7cac4/zentral/contrib/santa/api_views.py#L25) base view.